src/main/resources/application.yaml

```diff
@@ -52,6 +52,26 @@ aad:
 unleash:
   url: "https://unleash.herokuapp.com/api/"
 
+token.x:
+  privateJwk: |
```

Comment:
Er dette nødvendig å ha her? Selv om det ikke er for prod, så er det ikke lurt å ha hemmeligheter på github. Eller er det kanskje fra fakedings?